src/spider/storage/MysqlStorage.cpp

```
@@ -0,0 +1,131 @@
+auto MySqlMetadataStorage::connect(std::string /*url*/, boost::uuids::uuid /*id*/) -> StorageErr {
+    return StorageErr{StorageErrType::Success, ""};
+}
+
+void MySqlMetadataStorage::close() {}
+
+auto MySqlMetadataStorage::initialize() -> StorageErr {
+    return StorageErr{StorageErrType::Success, ""};
+}
+
+auto MySqlMetadataStorage::add_task_graph(TaskGraph const& /*task_graph*/) -> StorageErr {
+    return StorageErr{StorageErrType::Success, ""};
```

cppcheck

[performance] Variable 'm_id' is assigned in constructor body. Consider performing initialization in initialization list.

(useInitializationList)